terraform/azure/main.tf

```
@@ -38,6 +52,90 @@ resource "azurerm_storage_account" "orsoarpadev" {
   }
 
   tags = {
-    environment = "dev"
+    environment = var.environment
+  }
+}
+
+resource "azurerm_app_service_plan" "arpa" {
+  name                = module.app_service_plan_name.result
+  location            = azurerm_resource_group.arpa.location
+  resource_group_name = azurerm_resource_group.arpa.name
+
+  sku {
+    tier = "Free"
+    size = "F1"
+  }
+
+  tags = {
+    environment = var.environment
+  }
+}
+
+resource "azurerm_app_service" "arpa" {
+  name                = module.app_service_name.result
+  location            = azurerm_resource_group.arpa.location
+  resource_group_name = azurerm_resource_group.arpa.name
+  app_service_plan_id = azurerm_app_service_plan.arpa.id
+  https_only = true
+
+  site_config {
+    dotnet_framework_version = "v5.0"
+    scm_type                 = "GitHub"
+    http2_enabled = true
+    health_check_path = "/health"
+  }
+
+  app_settings = {
+    "Logging:IncludeScopes" = var.backendconfig.logConfig.includeScopes
+    "Logging:LogLevel:Default" = var.backendconfig.logConfig.logLevel.default
+    "Logging:LogLevel:Microsoft" = var.backendconfig.logConfig.logLevel.microsoft
+    "Logging:LogLevel:Microsoft.Hosting.Lifetime" = var.backendconfig.logConfig.logLevel.microsoftHostingLifetime
+    "Logging:LogLevel:Microsoft.EntityFrameworkCore.Database.Command" = var.backendconfig.logConfig.logLevel.microsoftEntityFrameworkCoreDatabaseCommand
+    "EmailConfiguration:From" = var.backendconfig.emailConfig.from
+    "EmailConfiguration:SmtpServer" = var.backendconfig.emailConfig.smtpServer
+    "EmailConfiguration:Port" = var.backendconfig.emailConfig.port
+    "EmailConfiguration:Username" = var.backendconfig.emailConfig.userName
+    "EmailConfiguration:Password" = var.backendconfig.emailConfig.password
+    "EmailConfiguration:DefaultSubject" = var.backendconfig.emailConfig.defaultSubject
+    "JwtConfiguration:TokenKey" = var.backendconfig.jwtConfig.tokenKey
+    "JwtConfiguration:Issuer" = "https://localhost:5001" // ToDo: Take from app service / custom domain
+    "JwtConfiguration:Audience" = "https://localhost:5001" // ToDo: Take from app service / custom domain
+    "JwtConfiguration:AccessTokenExpiryInMinutes" = var.backendconfig.jwtConfig.accessTokenExpiryInMinutes
+    "JwtConfiguration:RefreshTokenExpiryInDays" = var.backendconfig.jwtConfig.refreshTokenExpiryInDays
+    "IdentityConfiguration:LockoutExpiryInMinutes" = var.backendconfig.identityConfig.lockoutExpiryInMinutes
+    "IdentityConfiguration:MaxFailedLoginAttempts" = var.backendconfig.identityConfig.maxFailedLoginAttempts
+    "IdentityConfiguration:EmailConfirmationTokenExpiryInDays" = var.backendconfig.identityConfig.emailConfirmationTokenExpiryInDays
+    "IdentityConfiguration:DataProtectionTokenExpiryInHours" = var.backendconfig.identityConfig.dataProtectionTokenExpiryInHours
+    "CorsConfiguration:AllowedOrigins:0" = azurerm_storage_account.orsoarpadev.primary_blob_endpoint
```

I think you renamed the service account resource to "arpa"...

so you have to change the value to 

```suggestion
    "CorsConfiguration:AllowedOrigins:0" = azurerm_storage_account.arpa.primary_blob_endpoint
```

---

fixed